package.json

```diff
@@ -23,14 +21,16 @@
     "lodash.get": "^4.4.2",
     "moniker": "^0.1.2",
     "mz": "^2.7.0",
+    "pickle": "git+https://github.com:gobinaris/pickle.git#1.0.0",
```

Comment:
please publish pickle as a public npm package and consume it accordingly here. `@binaris/pickle` (just switch it to public after the first publish).

Reply:
sure